frontend/src/lib/components/Papercups.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```

Comment:
I think this file needs to be removed :) You can do it locally (`git rm` and `git push`) or from this web interface